### ISSUE TITLE
Avi researches GitHub wiki experience

### ISSUE THREAD

**Issue by avidrucker** [knowledge-for-good/knowledgeforgood#199]
- [ ] Use repo wiki for approx. 30 min
- [ ] Write down feedback on experience
- [ ] Share experience with KFG

**Comment by avidrucker** [comment]
Marking as paused and closed for now.

TLDR: My initial thinking is that using wiki features to capture information is adding new sources of truth (going against "single source of truth" practice). That said, perhaps wiki features *could* offer increased accessibility for some collaborators. Is the perception or actuality of increased accessibility simply a matter of "simple vs easy" ala "Simple Made Easy" [Rich Hickey's talk on YouTube](https://www.youtube.com/watch?v=oytL881p-nQ)? I believe familiarity with wiki features could make them (seem) relatively easier to use for some people.

Here's a somewhat convoluted, yet I believe realistic, illustration and conjecture: To new users of GitHub, adding issues and adding files to GitHub may represent "new things to learn". Gaining knowledge of and familiar with these features therefore may require some effort. Effort can lead to discomfort, and "bam" we could have a "wall of awful" [Jessica @ How to ADHD on "The Wall of Awful"](https://www.youtube.com/watch?v=Uo08uS904Rg). This could lead to a vicious cycle of avoidance, I believe, if mindfulness (eg. asking oneself "Am I ready to do this now?", "Am I HALT'd?"), accountability (eg. check-ins & follow-ups), and accessibility (eg. achievable, micro-task how-to's), are not sufficiently present.

I think when we are sufficiently experienced and healthy, these challenges can be relatively easy. But, we are constantly learning new things here at KFG, and I believe that we can help many people learn with less stress, and greater efficiency & efficacy, by applying the above concepts of mindfulness, accountability, and accessibility. An example of where these three components helped me to achieve greater results despite adversity is [the AutoFocus experiment (GitHub repo)](https://github.com/avidrucker/autofocus-exp).
